pkg/trebuchet/server/apiserver/apiserver.go

```diff
@@ -3,7 +3,7 @@ package apiserver
 import (
 	apis_trebuchet "github.com/atlassian/voyager/pkg/apis/trebuchet"
 	"github.com/atlassian/voyager/pkg/apis/trebuchet/install"
-	"github.com/atlassian/voyager/pkg/apis/trebuchet/v1"
+	v1 "github.com/atlassian/voyager/pkg/apis/trebuchet/v1"
```

Comment:
?

Reply:
Something something go imports is starting to add these. cc @ash2k

Reply:
Yes, new version which we have to use because rules_go needs it. goimports and other things which rules_go needs are all in x/tools repo.